cypress/integration/v3/blocks/text/index.js

```diff
@@ -0,0 +1,54 @@
+
+/**
+ * External dependencies
+ */
+import {
+	assertBlockExist, blockErrorTest,
+} from '~stackable-e2e/helpers'
+
+export {
+	blockExist,
+	blockError,
+	typeContent,
+	pressEnterKey,
+}
+
+function blockExist() {
+	it( 'should show the block', assertBlockExist( 'stackable/text', '.stk-block-text' ) )
+}
+
+function blockError() {
+	it( 'should not trigger block error when refreshing the page', blockErrorTest( 'stackable/text' ) )
+}
+
+function typeContent() {
+	it( 'should allow typing in the block', () => {
+		cy.setupWP()
+		cy.newPage()
+		cy.addBlock( 'stackable/text' )
+
+		cy.typeBlock( 'stackable/text', '.stk-block-text__text', 'Lorem ipsum dolor sit amet.', 0 )
+			.assertBlockContent( '.stk-block-text__text', 'Lorem ipsum dolor sit amet.' )
+	} )
+}
+
+function pressEnterKey() {
+	it( 'should test clicking the enter key in text block', () => {
```

Comment:
Also add a text for adding other blocks using `/`

Reply:
for text block

- Add tests for pressing backspace in text blocks